extensions/attention-stream/scripts/integration-test.sh

```diff
@@ -12,9 +12,13 @@ XVFB_RUN() {
 }
 
 # NOTE Firefox only supports manifest v2, re-add this when we have support and/or it ships manifest v3.
-#echo "Testing Firefox headless with extension"
```

Comment:
I just realized we never re-enabled these after the move to the monorepo - I filed https://github.com/mozilla-rally/rally/issues/199 to track, for now I just updated the commented-out commands so they should work when we do enable things.